csharp/ql/src/experimental/CWE-099/TaintedWebClient.ql

```diff
@@ -0,0 +1,25 @@
+/**
+ * @name Uncontrolled data used in a WebClient
+ * @description The WebClient class allow developers to request resources,
+ *              accessing resources influenced by users can allow an attacker to access local files.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
+ * @id cs/webclient-path-injection
+ * @tags security
+ *       external/cwe/cwe-099
+ *       external/cwe/cwe-022
```

Comment:
22 can be removed, as you have the more specific 23 and 36.

Reply:
Make sense, fixed.